index.html

```diff
@@ -0,0 +1,7 @@
+<div style="color: #35393B; margin-top: 100px; text-align: center; font-family: HelveticaNeue-Light, sans-serif;">
```

Comment:
Oops ! Good catch this is an accidental insert from my docker install troubleshooting. Will delete.